zio-schema/shared/src/main/scala/zio/schema/codec/BinaryCodec.scala

```diff
@@ -0,0 +1,16 @@
+package zio.schema.codec
+
+import zio.Chunk
+import zio.stream.ZPipeline
+
+trait BinaryCodec extends Codec[Chunk[Byte], Byte] {
+
+  type BinaryEncoder[A] = Encoder[Chunk[Byte], Byte, A]
```

Comment:
Probably best to put these type aliases in the companion object since they are standalone.